.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!---
+Verify first that your issue/request is not already reported on GitHub.
+
+This issue tracker is only for the bug reports, for different issues see:
```

Comment:
You can remove "the". Also, I'd replace "different" with "other".

Reply:
Done.